hokusai/production.yml

```diff
@@ -63,6 +63,78 @@ spec:
                 values:
                 - foreground
 
+
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: metaphysics-web-green
+spec:
+  replicas: 2
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: metaphysics
+        layer: application
+        component: web
+      name: metaphysics-web-green
+      namespace: default
+    spec:
+      containers:
+      - name: metaphysics-web-green
+        env:
```

Comment:
These environment variables override anything pulled in via `envFrom` below...

Reply:
Makes sense 👍